src/app/shared/services/app-update/app-update.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from "@angular/core";
+import { AppUpdate, AppUpdateInfo, AppUpdateAvailability } from "@capawesome/capacitor-app-update";
+
+@Injectable({
+  providedIn: "root",
+})
+export class AppUpdateService {
```

Comment:
nit(blocking)
Would be good for the to extend our patter of sync/async service and define any init methods like other services.